sopel/bot.py

```diff
@@ -311,6 +315,19 @@ def has_plugin(self, name):
         """Tell if the bot has registered this plugin by its name"""
         return name in self._plugins
 
+    def get_plugin_meta(self, name):
+        """Get info about a registered plugin by its name
+
+        :param str name: name of the plugin about which to get info
+        :return: the plugin's metadata
```

Comment:
It may be good to refer to where the contents of the metadata dict are defined/documented (anywhere other than https://github.com/sopel-irc/sopel/blob/e9ca7e1b163375aef62e8345276f12213eb29165/sopel/plugins/handlers.py#L93 ?)

Reply:
There's no other place (plugin metadata is new in 7.0/master), but I can link to the abstract handler's method's docstring (done in force-push).